CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Handover RESTMapper object to helmclient to reduce the number of REST API calls.
```

Comment:
```suggestion
- Pass RESTMapper to helmclient to reduce the number of REST API calls.
```

Could we go with this as its a bit shorter?

Reply:
Looks better, Thanks!